### PR TITLE
Teams page

### PR DESCRIPTION
**Pull request by dcts**
Implemented a team page with the following features:
- For each contributor, show: (1) telegram username, (2) image (3) social links (4) area of work (5) OG contributor status (6) in how many phases the contributor was active
- Show currently active contributors (anyone that contributed in the last 3 phases)
- Allow filtering by "active", "past" and "all"
- Allow filtering by area of work
- Link to more info on how to register as contributor / start contributing
- Mobile responsive

### CODE REVIEW

File: src/app/team/page.tsx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+import { SkeletonRectangle } from "components/Skeleton";
+
+export default function Team() {
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    dispatch(fetchTeamState());
+  }, [dispatch]);
+
+  return (
+    <div className="bg-[#141414] grow flex items-center justify-center pt-10">
```
Comment (nguvictor) [inline src/app/team/page.tsx:24]: Can we use dexter-grey-dark?
found in https://github.com/DeXter-on-Radix/website/blob/main/tailwind.config.js

File: src/app/team/page.tsx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+function ActivityStatusToggle({ filter }: { filter?: ActivityStatus }) {
+  const t = useTranslations();
+  const dispatch = useAppDispatch();
+  const { activityStatusFilter } = useAppSelector((state) => state.teamSlice);
+  const label = filter === undefined ? "all" : filter;
+  const isActive = activityStatusFilter === filter;
+  return (
+    <div
+      className={`cursor-pointer inline-block mx-1 mt-3 px-4 py-2 bg-[#232629] opacity-60 rounded-badge hover:opacity-100 ${
```
Comment (nguvictor) [inline src/app/team/page.tsx:72]: Can we use base-100 instead of #232629

File: src/app/team/page.tsx [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+
+function ExpertiseToggle({ filter }: { filter?: Expertise }) {
+  const t = useTranslations();
+  const dispatch = useAppDispatch();
+  const { expertiseFilter } = useAppSelector((state) => state.teamSlice);
+  const label = filter === undefined ? "all" : filter;
+  const isActive = expertiseFilter === filter;
+  return (
+    <div
+      className={`cursor-pointer inline-block mx-1 mt-3 px-4 py-2 bg-[#232629] opacity-60 rounded-badge hover:opacity-100 ${
```
Comment (nguvictor) [inline src/app/team/page.tsx:92]: base-100

File: src/app/team/page.tsx [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+        </>
+      )}
+    </div>
+  );
+}
+
+function ContributorCard({ contributor }: { contributor: Contributor }) {
+  const t = useTranslations();
+  return (
+    <div className="w-[250px] h-[120px] bg-[#232629] rounded-2xl m-2 p-4 relative">
```
Comment (nguvictor) [inline src/app/team/page.tsx:148]: Can we use bg-base-100 instead of 232629

File: src/app/team/page.tsx [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+  return (
+    <a className="mr-1 cursor-pointer" href={url} target="_blank">
+      {iconHtml}
+    </a>
+  );
+}
+
+function Badge({ text }: { text: string }) {
+  const color =
+    text === "OG" ? "bg-[#00ca92] text-black font-bold" : "bg-[#191B1D]";
```
Comment (nguvictor) [inline src/app/team/page.tsx:250]: bg-base-200 ?

**Review by nguvictor — DISMISSED** [review]

File: src/app/team/page.tsx (hunk 1 above)
Comment (dcts) [reply src/app/team/page.tsx]: fixed!

File: src/app/team/page.tsx (hunk 2 above)
Comment (dcts) [reply src/app/team/page.tsx]: fixed!

File: src/app/team/page.tsx (hunk 3 above)
Comment (dcts) [reply src/app/team/page.tsx]: fixed!

File: src/app/team/page.tsx (hunk 5 above)
Comment (dcts) [reply src/app/team/page.tsx]: fixed!

File: src/app/team/page.tsx (hunk 4 above)
Comment (dcts) [reply src/app/team/page.tsx]: fixed!

**Review by nguvictor — APPROVED** [review]

**Pull request merged**